src/components/Button/ButtonStyles.ts

```diff
@@ -0,0 +1,58 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+
+import Colors from '../../util/colors';
+import { IButtonStyles } from 'office-ui-fabric-react/lib/components/Button/Button.types';
+import { ButtonColor } from './enums';
+
+const primaryColors: IButtonStyles = {
```

Comment:
instead of `primaryColors` and `secondaryColors`, maybe we could use names `primary` and `secondary` and replace all of our CSS with JS styles? that's the eventual goal, as we'll get more good stuff like RTL support and locale-specific fonts.

Reply:
Sounds good.